semver/parse_range.ts

```diff
@@ -5,8 +5,8 @@ import { OPERATOR_XRANGE_REGEXP, XRANGE } from "./_shared.ts";
 import { parseComparator } from "./_parse_comparator.ts";
 import { parseBuild, parsePrerelease } from "./_shared.ts";
 
-function isWildcard(id: string): boolean {
-  return !id || id.toLowerCase() === "x" || id === "*";
+function isWildcard(id: string | undefined): boolean {
```

Comment:
Please use the optional `?` assertion for option arguments instead of `undefined` union types. Ditto to other instances.

Reply:
ok, but `?` is only available for functions whose required parameters don't follow an optional one. Meaning, I can change it in `isWildcard()`, but not in `parseNumber()` (`_shared.ts`)